cmd/config/set_test.go

```diff
@@ -1,163 +1,78 @@
 	}
 }
 
 // Test Config Set Command Fails when an invalid key is provided
 func TestConfigSetCmd_InvalidKey(t *testing.T) {
-	// Create the command
-	rootCmd := cmd.NewRootCommand()
-
-	// Redirect stdout to a buffer to capture the output
-	var stdout bytes.Buffer
-	rootCmd.SetOut(&stdout)
-	rootCmd.SetErr(&stdout)
-
-	rootCmd.SetArgs([]string{"config", "set", "pingctl.bogus=true"})
-
-	// Execute the command
-	err := rootCmd.Execute()
+	err := testutils.ExecutePingctl("config", "set", "pingctl.invalid=true")
 	if err == nil {
-		testutils.PrintLogs(t)
-		t.Fatalf("Expected error for providing invalid key, Captured StdOut: %q", stdout.String())
+		t.Errorf("Expected error for providing invalid key")
 	}
 }
 
 // Test Config Set Command Fails when an invalid value type is provided
 func TestConfigSetCmd_InvalidValueType(t *testing.T) {
-	// Create the command
-	rootCmd := cmd.NewRootCommand()
-
-	// Redirect stdout to a buffer to capture the output
-	var stdout bytes.Buffer
-	rootCmd.SetOut(&stdout)
-	rootCmd.SetErr(&stdout)
-
-	rootCmd.SetArgs([]string{"config", "set", "pingctl.color=notboolean"})
-
-	// Execute the command
-	err := rootCmd.Execute()
+	err := testutils.ExecutePingctl("config", "set", "pingctl.color=invalid")
 	if err == nil {
-		testutils.PrintLogs(t)
-		t.Fatalf("Expected error for providing invalid value type, Captured StdOut: %q", stdout.String())
+		t.Errorf("Expected error for providing invalid value type")
 	}
 }
 
 // Test Config Set Command Fails when no value is provided
 func TestConfigSetCmd_NoValueProvided(t *testing.T) {
-	// Create the command
-	rootCmd := cmd.NewRootCommand()
-
-	// Redirect stdout to a buffer to capture the output
-	var stdout bytes.Buffer
-	rootCmd.SetOut(&stdout)
-	rootCmd.SetErr(&stdout)
-
-	rootCmd.SetArgs([]string{"config", "set", "pingone.export.environmentId="})
-
-	// Execute the command
-	err := rootCmd.Execute()
+	err := testutils.ExecutePingctl("config", "set", "pingctl.color=")
 	if err == nil {
-		testutils.PrintLogs(t)
-		t.Fatalf("Expected error for providing no value, Captured StdOut: %q", stdout.String())
+		t.Errorf("Expected error for not providing a value")
 	}
 }
 
 // Test Config Set Command Executes normally when too many arguments are provided
 func TestConfigSetCmd_TooManyArgs(t *testing.T) {
-	// Create the command
-	rootCmd := cmd.NewRootCommand()
-
-	// Redirect stdout to a buffer to capture the output
-	var stdout bytes.Buffer
-	rootCmd.SetOut(&stdout)
-	rootCmd.SetErr(&stdout)
-
-	rootCmd.SetArgs([]string{"config", "set", "pingctl.color=false", "pingctl.bogus=true", "pingctl.bogus2=false"})
-
-	// Execute the command
-	err := rootCmd.Execute()
+	err := testutils.ExecutePingctl("config", "set", "pingctl.color=false", "pingctl.color=true")
 	if err != nil {
-		testutils.PrintLogs(t)
-		t.Fatalf("Err: %s, Captured StdOut:\n%s", err.Error(), stdout.String())
+		t.Errorf("Expected error for too many arguments provided")
```

Comment:
I'm confused by the wording of this message vs the comment on the method. Is the error expected or not?

Reply:
Fixed!